lightning/src/chain/onchaintx.rs

```diff
@@ -552,7 +555,7 @@ impl<ChannelSigner: Sign> OnchainTxHandler<ChannelSigner> {
 		// Check if any pending claim request must be rescheduled
 		for (first_claim_txid, ref request) in self.pending_claim_requests.iter() {
 			if let Some(h) = request.timer() {
-				if height >= h {
+				if cur_height >= h {
```

Comment:
Hmmmm, should we swap the ordering of things in `update_claims_view` and first bump RBF transactions and only process the requests after that? I'm a bit hesitant to given it will probably mean a *ton* of test changes.

Reply:
See above comment about height collision, I would favor this ordering swap to avoid re-broadcasting. I believe, post-anchor that kind of defaulting internal API issue might force us to reserve fee-bumping reserve for nothing ?

Though it's not related to this PR, and we can keep thinking about it.

Reply:
I'm a bit confused about that comment, but I'd think the solution is to track outputs which were on chain and spent and *never* re-spend them, instead of re-spending them after ANTI_REORG_DELAY.

Reply:
Yeah that would works too to blacklist confirmed utxos so i think it's more an issue on how duplicate block processing is interfering with `update_claim_view()` list of calls.

Reply:
Oh, hmm, I guess I just misread your above comment. Let me try the ordering swap in a new PR? I don't really want to hold this/0.0.99 up to redo a ton of tests.